microsite/data/plugins/playlist.yaml

```diff
@@ -0,0 +1,9 @@
+---
+title: Playlist
+author: Phil Kuang
+authorUrl: https://github.com/kuangp
+category: Discovery
+description: Create, share, and follow custom collections of entities available in the Backstage catalog.
+documentation: https://github.com/backstage/backstage/tree/master/plugins/playlist
+iconUrl: img/playlist-logo.png
```

Comment:
Did you make this logo? Or where did it come from? Just need to make sure we can use it in the repo that's all

Reply:
hmm I found this on Google somewhere, don't remember where ... I can change this to something that's free to use, do you know if this logo is ok: https://www.flaticon.com/premium-icon/playlist_4081218?related_id=4081218&origin=tag